daprdocs/content/en/reference/api/metadata_api.md

```diff
@@ -1,21 +1,30 @@
 ---
 type: docs
-title: "Metadata API reference"
+title: "Sidecar Metadata API reference"
```

Comment:
No real need to add sidecar here, since all the API are for the sidecar. If need this can be added in an overview of the API section (which does not exist currently)
```suggestion
title: "Metadata API reference"
```